node/runtime/pangolin/src/migrations.rs

```diff
@@ -23,14 +23,32 @@ impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> Weight {
-	migration::move_pallet(b"BeefyGadget", b"MessageGadget");
-	<darwinia_ecdsa_authority::Authorities<Runtime>>::put(
-		frame_support::BoundedVec::try_from(vec![array_bytes::hex_into_unchecked(
-			"0x68898db1012808808c903f390909c52d9f706749",
-		)])
-		.unwrap(),
-	);
+	use frame_support::StorageHasher;
+
+	let from = AccountId::from([
+		100, 118, 109, 58, 0, 0, 0, 0, 0, 0, 0, 136, 9, 249, 179, 172, 239, 29, 163, 9, 244, 155,
+		90, 185, 122, 76, 15, 170, 100, 230, 174, 73,
+	]);
+	let to = AccountId::from([
+		100, 118, 109, 58, 0, 0, 0, 0, 0, 0, 0, 227, 249, 15, 111, 231, 199, 11, 31, 140, 190, 188,
+		52, 119, 4, 141, 79, 50, 230, 31, 7, 204,
+	]);
+
+	let _ = Kton::transfer_all(Origin::signed(from.clone()), Address::from(to.clone()), false);
+	if let Some(v) = migration::take_storage_value::<Balance>(
+		b"Ethereum",
+		b"RemainingKtonBalance",
+		&frame_support::Blake2_128Concat::hash(from.as_ref()),
+	) {
+		migration::put_storage_value(
```

Comment:
Suggest adding some assertions in the `pre_upgrade` and `post_upgrade`.

Reply:
You could push to this branch directly.